File: 4oy1 Muhriddin Sobirjonov/5masala.py
import  os
os.system("cls")

a, b, c, d = map(int,  input("4 ta son kiriting   ").split())

if  a % 2 == 0:
    a += 10
else:
    a *= 3  
if  b % 2 == 0:
    b += 10
else:
    b *= 3
if  c % 2 == 0:
    c += 10
else:
    c *= 3    
if  d % 2 == 0:
    d += 10
else:
    d *= 3  

print(a, b, c, d, sep="  ")

File: 3oy1 Muhriddin Sobirjonov/5masala.c
#include    <stdio.h>
#include    <stdlib.h>

void    ALMASH(int *son1, int *son2){
    int a = *son1;
    *son1 = *son2;
    *son2 = a;
}

int main(){
    system("cls");
    int son1, son2;
    printf("ikkita son kiriting:  ");
    scanf("%d%d", &son1, &son2);
    printf("son1 ==> %d\t son2 ==> %d\n", son1, son2);

    ALMASH(&son1, &son2);
    puts("\n");
    printf("son1 ==> %d\t son2 ==> %d\n", son1, son2);

    return 0;
}